subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -416,8 +416,8 @@ If set to `true` and debugging is enabled, Gradle will run the build with the so
 `-Dorg.gradle.debug.suspend=(true,false)`::
 When set to `true` and debugging is enabled, the JVM running Gradle will suspend until a debugger is attached. _Default is `true`._
 
-`-Dorg.gradle.daemon.debug=true`::
-Debug <<gradle_daemon.adoc#gradle_daemon, Gradle Daemon>> process.
+`-Dorg.gradle.daemon.debug=true` (deprecated)::
```

Comment:
:x: I would either do all the deprecation work in another PR or in this one but not add `(deprecated)` here in this PR and the rest in another.

Deprecating this would require coverage that asserts that if used the user gets a proper deprecation warning.